core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -175,9 +175,17 @@ public ManagedChannelImpl build() {
    * {@link ClientTransportFactory} appropriate for this channel.  This method is meant for
    * Transport implementors and should not be used by normal users.
    */
-  @Internal
```

Comment:
Did you intend to remove this internal annotation. Maybe you want to make it `@ExperimantalApi`?